tests/pytest/test_regression.py

```diff
@@ -150,6 +169,30 @@ def rms(x):
     assert rms_precip == pytest.approx(rms_column_water_source, rel=0.1)
 
 
+def checksum_file(path: str) -> str:
```

Comment:
why not use this for the 'docker' versions of the files as well? I guess you'd need to change all those regressions too then

Reply:
It would be nice to refactor the image-tests to do this at some point too.  Ultimately, what do you think about having all the tests run "natively" with this mechanism and e.g. injecting a variable that says what the "environment" is? We could then run `pytest --environment <ENV> tests/pytest` in parallel. I did a similar refactor in fv3gfs-fortran.